ArmA.Studio/Dialogs/WorkspaceSelectorDialogDataContext.cs

```diff
@@ -18,6 +19,12 @@ public class WorkspaceSelectorDialogDataContext : IDialogContext
         public string CurrentPath { get { return this._CurrentPath; } set { this._CurrentPath = value; this.OKButtonEnabled = !string.IsNullOrWhiteSpace(value); this.RaisePropertyChanged(); } }
         private string _CurrentPath;
 
+        public List<String> WorkSpaceListBox { get { return this._WorkSpaceListBox;  } set { this._WorkSpaceListBox = value; this.RaisePropertyChanged(); } }
+        private List<String> _WorkSpaceListBox;
+
+        public String WorkSpaceListBoxSelectedItem { get { return this._WorkSpaceListBoxSelectedItem; } set { this._WorkSpaceListBoxSelectedItem = value; this.RaisePropertyChanged();  this.WorkSpaceListBoxSelectionChanged(); } }
```

Comment:
Extra field WorkSpaceListBoxSelectedItem should be removed and the SelectedItem can be bound to `CurrentPath` directly